packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -121,7 +121,15 @@ export default {
             const { issues, isFulfillable } = responseData;
 
             // Can now log these errors inside the map if necessary
-            const detailedIssues = issues.map(issue => ({ ...checkoutIssues[issue.code], ...issue }));
+            const detailedIssues = issues.map(issue => {
+                const checkoutIssue = checkoutIssues[issue.code];
+
```

Comment:
I think we should log issues in here as per my comment on line 123? This way if a surprising error code does come down, we will have a way to find it and potentially add it to our own issues.